.github/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+*       @kayman-mk @pascalfrenz
+
+# license file shouldn't be changed and needs to be reviewed by lawyers
+LICENSE @kayman-mk
```

Comment:
suggestion: use the "Organization Defaults" team here

Reply:
done 👍🏻

Reply:
Sorry, I meant the License entry. Maybe we can use the QCM team for everything else.

Reply:
I completely missed that last LICENSE entry there, this makes more sense now 😄 
Done